src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/themes/AceTheme.java

```diff
@@ -59,7 +59,10 @@ public native final String getName()
    
    public native final String getUrl()
    /*-{
-      return this.url;
+      if (this.url.startsWith("/"))
```

Comment:
`startsWith()` is a relatively new addition; it became part of JavaScript with ES6: https://www.w3schools.com/jsref/jsref_startswith.asp and may be missing for users with RStudio Server on older browsers.